eungabi/src/commonMain/kotlin/com/easternkite/eungabi/navigation/EunGabiGraph.kt

```diff
@@ -20,28 +20,67 @@ import androidx.compose.runtime.Composable
 import com.easternkite.eungabi.utils.withScheme
 import com.eygraber.uri.Uri
 
+/**
+ * A navigation scheme, which is used to convert and parse a route to a destination.
+ */
 const val SCHEME = "navhost://"
 
+/**
+ * The navigation graph.
+ * It holds destinations to navigate and find them by route.
+ */
 class EunGabiGraph (
+    /**
+     * The start destination. It is added to the backStack of the [EunGabiController] initially.
+     */
     val startDestination: String,
+
```

Comment:
🚫 **[ktlint]** <standard:no-blank-line-in-list> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected blank line(s) in value parameter list
<!-- __reviewdog__:ChA1M2YyYzQxZGVmMWFkZGM4EgZrdGxpbnQ= -->